packages/core/src/common/externals/NodeJsExternals.ts

```diff
@@ -124,11 +119,6 @@ export const NodeJsExternals = getNodeJsExternals()
  * @returns A {@link fs.PathLike}.
  */
 function toFsPathLike(path: FsLocation): fs.PathLike {
-	if (path instanceof Uri) {
-		// Convert WHATWG URL to string so that it will be converted
-		// to Node.js URL by the next if-block.
-		path = path.toString()
-	}
```

Comment:
This PR also switches to use the built-in URL class when possible (#1763) so this conversion is no longer needed.